src/components/Combobox/Dynamic/DynamicCombobox.tsx

```diff
@@ -139,7 +139,14 @@ const DynamicComboboxInner = <T extends Option>(
           {endAdornment && typed && <Box>{endAdornment(value)}</Box>}
         </Box>
       </ComboboxWrapper>
-      <Portal asChild ref={refs.setFloating} style={floatingStyles}>
+      <Portal
+        asChild
+        ref={refs.setFloating}
+        style={{
+          ...floatingStyles,
```

Comment:
**Question:** can we add `zIndex: 1` to hook that returns `floatingStyles`? thanks to that we will be sure that all dropdowns will have proper index.